client-src/js-src/cs-client.js

```diff
@@ -200,20 +200,18 @@ class ChromeStatusClient {
   }
 
   // Approvals, configs, and review comments
-
-  getApprovals(featureId) {
-    return this.doGet(`/features/${featureId}/approvals`);
-  }
-
   setApproval(featureId, gateType, state) {
     return this.doPost(
         `/features/${featureId}/approvals`,
         {gateType: Number(gateType), state: Number(state)});
   }
 
   getVotes(featureId, gateId) {
+  if (gateId) {
```

Comment:
It looks like you need to indent this if-statement so that it is inside the function.